app/models/responsiblepeople/ResponsiblePerson.scala

```diff
@@ -141,10 +141,14 @@ case class ResponsiblePerson(personName: Option[PersonName] = None,
     Logger.debug(s"[ResponsiblePeople][isComplete] $this")
 
     this match {
+      case ResponsiblePerson(Some(_),Some(_),Some(_),Some(_),Some(_), _, _, Some(_),Some(_),Some(_), Some(pos),Some(_), _,Some(_),Some(_), _, _, true, _, _, _, otherBusinessSP)
+        if pos.startDate.isDefined & checkVatField(otherBusinessSP) & validateAddressHistory & ApplicationConfig.phase2ChangesToggle => true
+      case ResponsiblePerson(Some(_),Some(pName),None,Some(_),Some(_), _, _, Some(_),Some(_),Some(_), Some(pos),Some(_), _,Some(_),Some(_), _, _, true, _, _, _, otherBusinessSP)
+        if pos.startDate.isDefined & checkVatField(otherBusinessSP) & validateAddressHistory && !pName.hasPreviousName.get & ApplicationConfig.phase2ChangesToggle => true
       case ResponsiblePerson(Some(_),Some(_),Some(_),Some(_),Some(_), _, _, _,Some(_),Some(_), Some(pos),Some(_), _,Some(_),Some(_), _, _, true, _, _, _, otherBusinessSP)
-        if pos.startDate.isDefined & checkVatField(otherBusinessSP) & validateAddressHistory => true
+        if pos.startDate.isDefined & checkVatField(otherBusinessSP) & validateAddressHistory & !ApplicationConfig.phase2ChangesToggle => true
```

Comment:
Why use ApplicationConfig rather than AppConfig?

Reply:
AppConfig is the injectable version, and we couldn't find a way of injecting it without making masses of changes throughout the service - so we fell back to the ApplicationConfig which limited the impact

Reply:
Ok. Cool.